src/Modules/ConfigOptions.lua

```diff
@@ -42,6 +42,22 @@ local function banditTooltip(tooltip, mode, index, value)
 	end
 end
 
+local function bossSkillsTooltip(tooltip, mode, index, value)
+	local applyModes = { BODY = true, HOVER = true }
+	tooltip:Clear()
+	if applyModes[mode] then
+		tooltip:AddLine(14, [[
+^7Used to fill in defaults for specific boss skills if the boss config is not set
+
+Bosses' damage is modified by roll range configuration, defaulted at a 70% roll, at the normal monster level for your character level (capped at 85)
+Fill in the exact damage numbers if more precision is needed
+]])
+		if value.val ~= "None" then
+			tooltip:AddLine(14, '^7'..value.val..": "..data.bossSkills[value.val].tooltip)
+		end
+	end
+end
```

Comment:
```suggestion
Fill in the exact damage numbers if more precision is needed]])
		if value.val ~= "None" then
			tooltip:AddLine(14, '\n^7'..value.val..": "..data.bossSkills[value.val].tooltip)
		end
	end
end
```
This should avoids there being an unnecessary newline when no skill is selected.
![image](https://user-images.githubusercontent.com/31533893/213995542-d5c85070-64f0-405a-b13a-4df041eec32f.png)